api/package.json

```diff
@@ -40,7 +40,8 @@
     "sequelize": "^4.37.8",
     "sequelize-cli": "^4.0.0",
     "standard": "^12.0.1",
-    "winston": "^3.1.0"
+    "winston": "^3.1.0",
+    "ws": "^6.2.1"
```

Comment:
Don't we want to use something like socket-io? This could radically simplify things.

Reply:
I decided to use ws as it is simpler and lighter. socket-io is a very good library with a lot of features. I think, for the moment, we do not need all the features that provides. Maybe refactor on another PR ?

Reply:
Sounds good, let's make it an issue for another PR. I think refactoring with io would help eliminate ping/pong code and a significant amount of boilerplate that's in this PR, but since it's here now we might as well replace it later. I think you'll find that socket-io is a wonderful library and quite simple to use though -- don't be afraid to try it out.